Codes/Clean Data/getAggressiveness.R

```diff
@@ -0,0 +1,22 @@
+function(direction,
+         price,
+         mid,
+         ssquote,
+         indexSub,
+         string) {
+  

```

Comment:
I may ask something that I fell most annoying myself. :(  The docstring for the function. At less the meaning of argument. Trust me, I hate docstring as much as other modelers. But it's essential for other people to understand what you are doing in the function. Here is the R docstring standard: [https://cran.r-project.org/web/packages/docstring/vignettes/docstring_intro.html]